package.json

```diff
@@ -27,18 +27,18 @@
   },
   "homepage": "https://github.com/mosaicdao/mosaic-1#readme",
   "devDependencies": {
-    "assert": "^2.0.0",
-    "bn.js": "^5.0.0",
-    "chai": "^4.2.0",
-    "eslint": "^5.16.0",
-    "eslint-config-airbnb-base": "^13.1.0",
-    "eslint-plugin-import": "^2.17.3",
-    "ethlint": "^1.2.4",
-    "ganache-cli": "^6.4.4",
-    "mocha": "^6.2.0",
-    "openzeppelin-solidity": "^2.1.1",
-    "solium": "^1.2.4",
-    "truffle": "^5.0.0-beta.2",
+    "assert": "2.0.0",
+    "bn.js": "5.0.0",
+    "chai": "4.2.0",
+    "eslint": "5.16.0",
+    "eslint-config-airbnb-base": "13.1.0",
+    "eslint-plugin-import": "2.17.3",
+    "ethlint": "1.2.4",
+    "ganache-cli": "^6.7.0",
+    "mocha": "6.2.0",
+    "openzeppelin-solidity": "2.1.1",
+    "solium": "1.2.4",
+    "truffle": "5.0.0-beta.2",
```

Comment:
We can move out of beta here, for truffle latest release is now`5.1.0`.
Similarly, for `web3` we can move out of beta version, latest is `1.2.4`

Reply:
Updated the truffle version. But web3 version 1.2.4 is causing a test failure. Needed to investigate separately. Created ticket #29